### PR TITLE
feature/AQI-4634 added protobuf field Enterprise.DatasetDefinition.qu…

### PR DESCRIPTION
**Pull request by AlanLiu-AI**
…eryParameters

### CODE REVIEW

File: src/ONE.Models.CSharp/ONE.Models.CSharp.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-    <VersionPrefix>4.14.0</VersionPrefix>
+    <VersionPrefix>4.15</VersionPrefix>
     <!--For Feature packages update the VersionSuffix with alpha and an incremental number. E.g. alpha1, alpha2...-->
     <VersionSuffix></VersionSuffix>
```
Comment (AQI-DanG) [inline src/ONE.Models.CSharp/ONE.Models.CSharp.csproj:7]: Should we be doing a pre-release for this package?

Comment (AlanLiu-AI) [reply src/ONE.Models.CSharp/ONE.Models.CSharp.csproj:7]: We won't directly make functional change, which is a separate story, therefore, no meaningful to do a pre-release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 4.15.0
+### Added
+- Added EnterrpiseDatasetDefinition.queryParameters
```
Comment (KentMiller) [inline CHANGELOG.md:6]: Misspelled Enterprise

**Review by KentMiller — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 4.15.0
+### Added
+- Added EnterpiseDatasetDefinition.queryParameters
```
Comment (tomsmith-ai) [inline CHANGELOG.md:6]: should be Enterp**r**iseDatasetDefinition

File: src/ONE.Models.CSharp/ONE.Models.CSharp.csproj [hunk 4]
```diff
@@ -18,11 +18,9 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <PackageReleaseNotes>
-		Version 4.14.0
-		- Added new value to EnumDataSource
-		- Deprecated several existing values in EnumSamplingStatistic
-		- Added new values to EnumSamplingStatistic
-	</PackageReleaseNotes>
+        Version 4.15.0
+        - Added EnterpiseDatasetDefinition.queryParameters
```
Comment (tomsmith-ai) [inline src/ONE.Models.CSharp/ONE.Models.CSharp.csproj:22]: Again, Enterp**r**iseDatasetDefinition

**Review by tomsmith-ai — CHANGES_REQUESTED** [review]
a couple minor spelling errors, might as well deal with those.

**Review by tomsmith-ai — APPROVED** [review]
THanks Alan

**Pull request merged**